hxd/res/BDFFont.hx

```diff
@@ -0,0 +1,436 @@
+package hxd.res;
+
+#if (haxe_ver < 4)
+import haxe.xml.Fast in Access;
+#else
+import haxe.xml.Access;
+#end
+
+import hxd.BitmapData;
+import haxe.Int32;
```

Comment:
Is there a particular reason you need to explicitly specify it's an Int32? If no, better to just use `Int`, which is a core type and not an abstract with overhead.

Reply:
It holds an ARGB value and I wanted to be sure Int was wide enough. If Int is always at least 32 bit then I'm happy to change to Int?

Reply:
Unless Haxe (and Heaps) somehow starts to support 16-bit target with standard int being less than 32-bit in size - it is guaranteed to be at least of that size.